### DISCUSSION TITLE
Sort order for related resources?

### DISCUSSION THREAD

**Discussion by ikreymer** [metacontroller/metacontroller#793] [Q&A]
I was wondering if there is a defined sort order for the flat lists of resources returned by CompositeController, in particular, the list of resources returned in the `related` list for each object type as enabled via the customize hook.

I would like to sort the resources based on creation time (using `createdTimestamp`), however, from testing it looks like the resources are already in this order every time!

Is this something that is guaranteed (resources sorted in creation order) or is the sort order of related resources not defined/accidentally happens to work. If the former, then can drop the sorting step in my code, otherwise will need to sort.

Also, wondering if this would apply to the list of `children` as well.

**Comment by grzesuav** [comment]
hi @ikreymer , I do not believe it is guaranteed, it just the order it is returned from caches. I would not assume it is ordered
